src/tabnineChatWidget/workspaceCommands/commandExecutors/index.ts

```diff
@@ -0,0 +1,3 @@
+export interface CommandExecutor {
```

Comment:
can be T , instead of string[]

Reply:
i don't see a reason to make it generic, as eventually all the data must be `string[]` in order to get into the prompt, so this way I'm enforcing the executors to serialize it correctly